articles/quickstart/native/android/_includes/_login.md

```diff
@@ -70,12 +99,7 @@ The browser will redirect to your application with the authentication result. Be
 </manifest>
 ```
 
-There are many options to customize the authentication using the `WebAuthProvider` builder. Make sure to check them [here](/libraries/auth0-android#implementing-web-based-auth).
+There are many options to customize the authentication with the `WebAuthProvider` builder. You can read about them in the [Auth0 SDK for Android documentation](/libraries/auth0-android).
 <div class="phone-mockup">
   <img src="/media/articles/native-platforms/android/login-android.png" alt="Mobile example screenshot" />
-</div>
-
-
-## Hosted Login Page vs Embedded Login
```

Comment:
@aaguiarz do we want to remove this section? Is now mentioned here in the "start" step, which IMO can get more attention than before (which might be bad) https://github.com/auth0/docs/pull/5017/files#diff-4bdb4aedb0f37d8c3116bbdfcff71d65R6

Reply:
Yes, we should remove the section, it is already mentioned. We are showing it in the start steps in all SPA quickstarts, so it's fine.